skymp5-server/cpp/unit/EspmTest.cpp

```diff
@@ -6,8 +6,18 @@
 extern espm::Loader l;
 
 // These tests depend on the most recent files shipped with Skyrim SE.
-// See README.md in project root for details.
-// TODO(#107): make sure hashes of files match expected ones.
+// See README.md in project root for details
```

Comment:
Original line had `.` in the end, please return it so we can avoid unnecessary diff